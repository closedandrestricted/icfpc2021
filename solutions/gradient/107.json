{"vertices":[[47,82],[52,82],[65,94],[61,96],[47,64],[52,71],[56,70],[81,99],[52,67],[54,66],[46,64],[70,92],[46,61],[56,114],[98,118],[83,83],[60,118],[84,87],[47,58],[66,85],[99,108],[77,102],[70,84],[82,90],[114,99],[82,105],[101,94],[54,66],[53,69],[55,75],[71,93],[80,83],[95,91],[79,88],[66,94],[77,83],[59,78],[63,68],[81,91],[115,86],[78,116],[67,67],[89,126],[79,120],[81,81],[100,70],[125,78],[93,122],[83,80],[74,58],[79,69],[101,125],[84,70],[134,86],[104,128],[82,54],[106,82],[105,122],[110,132],[108,119],[88,68],[88,61],[125,82],[87,65],[92,83],[104,39],[125,98],[110,89],[105,31],[90,75],[120,102],[101,83],[91,87],[101,94],[84,86],[141,133],[134,118],[77,89],[91,36],[131,116],[84,39],[94,101],[97,113],[91,45],[93,102],[69,80],[95,57],[90,104],[96,109],[67,86],[94,51],[86,110],[95,62],[85,102],[96,73],[84,100],[91,104],[81,102],[107,105],[111,102],[94,71],[91,70],[53,86],[49,85],[105,74],[100,102],[116,110],[111,74],[106,98],[118,116],[42,93],[39,91],[40,104]]}