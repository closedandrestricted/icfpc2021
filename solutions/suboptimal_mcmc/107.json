{"vertices":[[47,82],[52,80],[65,93],[61,94],[48,60],[52,69],[56,69],[90,82],[52,65],[54,64],[46,62],[78,77],[46,59],[56,112],[99,116],[88,84],[60,116],[84,85],[47,56],[69,81],[99,106],[78,100],[70,85],[82,88],[115,97],[83,103],[102,94],[54,64],[53,67],[55,73],[72,94],[80,81],[96,91],[79,86],[67,94],[77,81],[59,76],[63,66],[81,89],[115,84],[81,114],[67,66],[91,125],[80,118],[84,80],[103,68],[125,78],[93,120],[85,78],[77,57],[79,69],[101,123],[84,68],[134,86],[104,126],[84,53],[107,82],[105,120],[110,130],[108,117],[88,68],[88,59],[126,91],[87,63],[94,80],[104,37],[126,97],[110,87],[105,29],[90,73],[121,100],[101,82],[91,86],[102,94],[84,84],[141,131],[134,116],[77,87],[91,34],[131,114],[84,37],[94,101],[97,111],[92,43],[93,100],[69,78],[95,55],[90,102],[96,107],[67,84],[92,49],[86,108],[95,60],[85,100],[97,68],[84,98],[89,105],[81,100],[107,103],[111,100],[94,69],[91,68],[51,79],[49,83],[105,72],[101,102],[116,108],[110,75],[106,96],[116,115],[42,91],[40,88],[33,98]]}