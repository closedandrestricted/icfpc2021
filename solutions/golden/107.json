{"vertices":[[46,80],[51,80],[50,100],[52,96],[43,67],[52,70],[56,69],[85,87],[49,69],[50,67],[45,64],[80,75],[44,61],[61,113],[98,117],[78,82],[60,119],[81,85],[53,51],[69,96],[98,107],[77,102],[70,92],[84,77],[115,99],[82,104],[103,93],[57,61],[54,62],[55,74],[88,98],[80,83],[97,91],[79,88],[87,103],[77,83],[55,69],[63,68],[82,90],[113,85],[83,115],[67,68],[96,117],[80,118],[82,82],[94,96],[125,89],[93,122],[83,80],[73,61],[79,72],[101,125],[84,70],[138,87],[104,128],[79,55],[97,83],[105,122],[110,132],[108,119],[71,69],[87,60],[129,91],[87,64],[105,71],[104,39],[120,117],[113,87],[105,31],[93,73],[121,123],[102,87],[92,85],[110,97],[81,86],[142,144],[136,128],[74,84],[91,36],[138,125],[84,39],[120,96],[97,113],[91,45],[93,102],[67,93],[96,57],[90,104],[96,109],[64,87],[94,51],[87,108],[97,62],[82,101],[102,70],[84,100],[82,92],[81,102],[106,101],[111,101],[94,70],[91,70],[50,88],[46,86],[102,79],[97,97],[107,112],[104,84],[102,91],[101,115],[40,94],[37,92],[40,104]]}